src/features/rewards/siteBanner/index.tsx

```diff
@@ -23,7 +23,6 @@ import {
   StyledSocialItem,
   StyledSocialIcon,
   StyledOption,
-  StyledIconRecurring,
```

Comment:
Please remove this and `RefreshIcon` from siteBanner/style.ts then we are good :)

Reply:
Done.